src/Generator.Rewrite/CountAttribute.cs

```diff
@@ -1,9 +1,23 @@
 ﻿namespace OpenTK.Rewrite
 {
+    /// <summary>
+    /// A copy of OpenTK.CountAttribute. Only for internal use.
+    /// </summary>
     internal class CountAttribute
```

Comment:
I think we'd rather reference the OpenTK subproject that defines these attributes and refer to that instead.

Reply:
I tried to hold off on referencing other projects, but I guess it would be fine either way. Sure, I'll change it

Reply:
Yeah, that's true. Disregard original comment.